src/record.js

```diff
@@ -6,14 +6,35 @@
 
 // @flow
 
-import int53 from "int53";
-
 import { extractFields, extractTime } from "./fields";
 import { MessageReader } from "./MessageReader";
 import type { Time } from "./types";
 
-const readUInt64LE = (buffer: Buffer) => {
-  return int53.readUInt64LE(buffer, 0);
+const readUInt32LE = (a: Uint8Array, offset: number) => {
+  const view = new DataView(a.buffer);
+  return view.getUint32(a.byteOffset + offset, true);
+};
+
+const readInt32LE = (a: Uint8Array, offset: number) => {
+  const view = new DataView(a.buffer);
```

Comment:
Slightly worried about constructing a `DataView` for every integer field. Do you know if it costs us much?